gensim/models/fasttext.py

```diff
@@ -987,7 +1179,7 @@ def _load_fasttext_format(model_file, encoding='utf-8', full_model=True):
     encoding : str, optional
         Specifies the file encoding.
     full_model : boolean, optional
-        If False, skips loading the hidden output matrix.  This saves a fair bit
+        If False, skips loading the hidden output matrix. This saves a fair bit
         of CPU time and RAM, but prevents training continuation.
```

Comment:
somehow highlight `prevents training continuation.` (make it bold maybe?)

Reply:
Bolded